CMakeLists.txt

```diff
@@ -638,7 +638,16 @@ include(InstallHelpers)
 add_glob_targets(
   TARGET lintsh
   COMMAND ${SHELLCHECK_PRG}
+  FLAGS -x -a --exclude=2086
   FILES scripts/vim-patch.sh
+    scripts/download-unicode-files.sh
+    scripts/git-log-pretty-since.sh
+    scripts/release.sh
+    scripts/update_terminfo.sh
+    ci/before_cache.sh
+    ci/before_script.sh
+    ci/install.sh
+    ci/run_tests.sh
```

Comment:
can we make this an exclude instead of include? or we still have too many unchecked scripts?

Reply:
Yeah, it's not pretty. I'll come up with something better.